src/executables/csql.c

```diff
@@ -79,6 +83,14 @@
 #define strlen(s1)  ((int) strlen(s1))
 #endif /* defined (SUPPRESS_STRLEN_WARNING) */
 
+#define DB_NAME_LEN MAXLOGNAME
+#define FREE_MEM(PTR)           \
+        do {                    \
+          if (PTR) {            \
+            free(PTR);          \
+          }                     \
+        } while (0)
+
```

Comment:
It is good to define FREE_MEM macro, but csql already uses macro '**free_and_init** ()'  with the same meaning (defined in base/memory_alloc.h). 
How about not defining FREE_MEM and just using the **free_and_init**() macro like any other csql code?

Reply:
I encounter the error when the PTR is set to NULL.
(I mean I also user free_and_init() and then error occured)